app/assets/javascripts/admin/views/editions/_form.js

```diff
@@ -121,6 +91,147 @@
       })
     },
 
+    toggleWorldNewsStoryVisibility: function toggleWorldNewsStoryVisibility() {
+      // only toggle fields when news article type is editable
+      if ( $( "select#edition_news_article_type_id" ).length == 0 ) return;
+
+      var nonWorldNewsStoryFieldsets = this.findFieldsets(this.nonWorldNewsStoryFieldSelectors())
+      var worldNewsStoryFieldsets = this.findFieldsets(this.worldNewsStoryFieldSelectors())
+
+      var _this = this
```

Comment:
I think we should probably choose a different name here as we conventionally use `_var_name` to indicate that a variable isn't used. Maybe we should go with `var that = this`